### DISCUSSION TITLE
my Landscape Theme

### DISCUSSION THREAD

**Discussion by 1080papaya** [mathoudebine/turing-smart-screen-python#108] [Themes]
preview - 
![image](https://user-images.githubusercontent.com/120036534/208128675-897f60cd-5647-40b7-b074-b56b67e775dd.png)

![template_turingscreen](https://user-images.githubusercontent.com/120036534/208127439-d2332836-498c-46bb-a3a2-08fe42df4eeb.png)

---------------------
RENAME .txt to .YAML
[theme.txt](https://github.com/mathoudebine/turing-smart-screen-python/files/10246828/theme.txt)

I use a batch script combined with Playnite (video game library manager) to automatically switch profiles when i load a game, here are some other flavours for this theme.

Let me know if you would like the .PSD to use for yourself.

Edit - i just realized i'll have to remove the GPU/CPU text that i have on these photos, I'll recreate them by drawing static text instead and re-upload at a later date! 

![3](https://user-images.githubusercontent.com/120036534/208126535-0b831f51-b932-4dbf-b03d-fac6fa74a3e4.png)
![fate](https://user-images.githubusercontent.com/120036534/208126549-7d84d534-a1a4-41b3-ab8c-0341baa8ea47.png)
![background](https://user-images.githubusercontent.com/120036534/208126669-1f728145-48fd-4571-9412-b4f67d200835.png)
![background](https://user-images.githubusercontent.com/120036534/208126703-71074b5e-ee08-4a81-a8a3-5a0ebd515a26.png)
![background](https://user-images.githubusercontent.com/120036534/208126735-a25213f1-f97a-47e3-a7ee-f7424dfe24be.png)
![background](https://user-images.githubusercontent.com/120036534/208126747-7b88aa0e-9196-4a06-bc0b-88a36685493c.png)

**Comment by mathoudebine** [comment]
Thanks for sharing this nice theme with us!

**Comment by Caskexe** [comment]
@1080papaya this is awesome. Would love the PSD if you're still sharing :)

**Comment by 1080papaya** [comment]
I'll see if I still have the psd. My hard drive recently failed so I'm not
even sure if I backed it up :(

**Comment by mizukika** [comment]
Hi, can you please give PSD for this )

**Comment by 1080papaya** [comment]
Let me check my HDD if I still have them around

**Comment by 1080papaya** [comment]
i may have actually lost the original psd, ill check this week to see if i
might have created a backup of it. sorry guys

**Reply by mizukika** [reply]
Sad, I'll hope you still have a backup :)

**Comment by isayx3** [comment]
Thank You! I used this as a template to customize my own. 
![background](https://github.com/user-attachments/assets/3de9374a-0c36-4748-affb-7c48395d2c4d)

**Reply by mizukika** [reply]
Can you give please teamplate

**Reply by isayx3** [reply]
I've updated it since. Keep in mind you can always go into the yaml file and shift things around so things align better.

[Template PSD](https://drive.google.com/file/d/16fxQLCueEztjBKtu3SORRsyZDK0DUuPd/view?usp=sharing)

**Reply by mizukika** [reply]
thank you very much!
